hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -451,3 +451,16 @@ message ContainerBalancerConfigurationProto {
     required bool shouldRun = 18;
     optional int32 nextIterationIndex = 19;
 }
+
+message DiskBalancerConfigurationProto {
+    optional string threshold = 1;
+    optional string diskBandwidth = 2;
+    optional int32 parallelThread = 3;
+}
+
+message DatanodeDiskBalancerInfoProto {
+    optional DatanodeDetailsProto node = 1;
```

Comment:
This should be required instead of optional.

Reply:
> node, diskBalanderRunning and currentVolumeDensitySum should be required instead of optional.

Changed "node" and "currentVolumeDensitySum" to required, but kept "diskBalancerRunning" as optional since "Report" command won't return the running status.

> use double for threashold, and uint64 for diskBandwidth, same for DiskBalancerConfigurationProto.

Used double for threshold and diskBandwidth (seems double is better?)

Reply:
Please use uint64 instead. diskBandwidth is a number without fraction.  We can change it in next PR.